.buildkite/commands/run-tests.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash -eu
+
+SIMULATOR_NAME=$1
+
+xcodebuild clean test \
+  -project 'ScreenObject.xcodeproj' \
+  -scheme 'ScreenObject' \
+  -destination "platform=iOS Simulator,name=$SIMULATOR_NAME,OS=14.5"
```

Comment:
I decided to:

1. Keep this simple and don't integrate Fastlane just to run the tests, nor `xcpretty` or `xcbeautify` to format the output. All of those options would require extra dependency and CI setup that doesn't seem granted at this time
2. Hardcode the iOS version here, rather than pass it to the script, so we have a single source of truth. This makes the syntax a bit less clear in the pipeline, though. Given the scope of this project, I think we can deal with it for the moment. I can also see a future in which we might want to run tests against multiple iOS version, if we get to it, we can start passing the value in the same way as we do with `$SIMULATOR_NAME`.